Tests/XCUITests/IntegrationTests.swift

```diff
@@ -198,13 +198,13 @@ class IntegrationTests: BaseTestCase {
         navigator.nowAt(SettingsScreen)
         navigator.goto(LoginsSettings)
         waitForExistence(app.buttons.firstMatch)
-        app.buttons["Continue"].tap()
 
         let springboard = XCUIApplication(bundleIdentifier: "com.apple.springboard")
         let passcodeInput = springboard.secureTextFields["Passcode field"]
         passcodeInput.tap()
         passcodeInput.typeText("foo\n")
 
+        app.buttons["Continue"].tap()
```

Comment:
Note: The appearance of "Continue" button may change. Waiting for clarification.

Reply:
Let's keep an eye on this tests while that is clarified and land this PR to see if tests work better

Reply:
Note that some tests may still fail due to Firefox on the simulator crashing randomly. I've caught the tests failing due to Firefox quitting midway.

Reply:
if that's the case there might be some crashes we will need to try to find a report. For the login tests they will fail because of the biometric info screen, I'm sure that's the reason